cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskPlanner.java

```diff
@@ -297,10 +298,12 @@ public Collection<ExecutionTask> remainingLeadershipMovements() {
    *
    * @param leadershipConcurrencyByBrokerId the leadership movement concurrency of each broker
    * @param clusterLeadershipMovementConcurrency the allowed movement concurrency of the whole cluster
+   * @param executionConcurrencyManager the execution concurrency manager
    * @return The leadership movement tasks.
    */
   public List<ExecutionTask> getLeadershipMovementTasks(Map<Integer, Integer> leadershipConcurrencyByBrokerId,
-                                                        int clusterLeadershipMovementConcurrency) {
+                                                        int clusterLeadershipMovementConcurrency,
```

Comment:
Since the executionConcurrencyManager has all the information needed, let's change the method signature to getLeadershipMovementTasks(ExecutionConcurrencyManager executionConcurrencyManager).

Reply:
Thanks @CCisGG, you are right, that makes a lot more sense, just updated the PR to address this